Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Troll colored emblems + co a

## Changelog:
Added a bunch of landed Coat of Arms and a few unique dynastic Coat of Arms for Trolls


## Developer changelog:

- 45 new .dds files, ce_troll1 - ce_troll13 intended for random title generation
- commits for the new coat of arms in 00_wc_landed_titles.txt and wc_dynasties.txt

# How to test:
boot up ur game and look at the following titles to see if flags and dynasties work:

- [ ] Empire of Amani
- [ ] Kingdom of Zul'Aman
- [ ] Duchy of Zul'Aman
- [ ] Duchy of Mossflayer
- [ ] Empire of Zandalari
- [ ] Kingdom of Zandalar
- [ ] Duchy of Dazar'alor
- [ ] Empire of Farraki
- [ ] Kingdom of Zul'Farrak
- [ ] Duchy of Zul'Farrak
- [ ] Empire of Drakkari
- [ ] Kingdom of Zul'Drak
- [ ] Kingdom of Drak'tharon
- [ ] Duchy of Gundrak
- [ ] Empire of Gurubashi
- [ ] Kingdom of Zul'Gurub
- [ ] Duchy of Zul'Gurub
- [ ] Duchy of Shatterspear
- [ ] Kingdom of Hinterlands
- [ ] Duchy of Vilebranch
- [ ] Duchy of Witherbark
- [ ] Kingdom of Frostmane
- [ ] Duchy of Bloodscalp
- [ ] Duchy of Smolderthorn
- [ ] Duchy of Skullsplitter
- [ ] Kingdom of South Islands
- [ ] Duchy of Darkspear
- [ ] Dynasty of Malakk Jin (Ruler of Zul'Drak)
- [ ] Dynasty of Rastakhan Khan (Ruler of Zandalari)
- [ ] Dynasty of Sen'jin Jin (Ruler of Darkspear)
- [ ] Dynasty of Ukorz Sandscalp (Ruler of Zul'Farrak)

Also: Check if this commit conflicts with previous conflict. Possible problem maybe if there are 2 checks for the same title in 00_wc_landed_titles.txt
